php/class-fieldmanager-field.php

```diff
@@ -979,8 +983,49 @@ public function add_page_form( $uniqid ) {
 	 * @param int $parent Only show this field on child terms of this parent term ID
 	 */
 	public function add_term_form( $title, $taxonomies, $show_on_add = true, $show_on_edit = true, $parent = '' ) {
+		_deprecated_function( __METHOD__, 'Fieldmanager-1.0.0-beta.3', 'Fieldmanager_Field::add_term_meta_box()' );
```

Comment:
Also, see recent `wp_title()` [debacle](https://core.trac.wordpress.org/changeset/35624) to see the impact of calling `_deprecated_function()` too soon.

I think `add_term_meta_box()` should exist for a couple to few WordPress releases before you formally deprecate `add_term_form()`. Otherwise, developers will need to have a lot of conditionals in their code if they're working against trunk locally, and using an outdated version of WordPress in production (cough cough VIP)